spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -88,6 +88,8 @@ Google Cloud Spanner API, if different from the ones in the
 | `spring.cloud.gcp.spanner.credentials.scopes` |
 https://developers.google.com/identity/protocols/googlescopes[OAuth2 scope] for Spring Cloud GCP
 Cloud Spanner credentials | No | https://www.googleapis.com/auth/spanner.data
+| `spring.cloud.gcp.spanner.createTableStatementsCascadeOnDelete` | If `true`, then schema statements generated by `SpannerSchemaUtils` for tables
```

Comment:
I guess we haven't finalized the name of the property yet.
Will it be `createInterleavedTableDdlOnDeleteCascade`?